### PR TITLE
Update install-k8s-addon-eks.rst

### PR DESCRIPTION
**Pull request by chentaow-splunk**
1. We want to also make sure the Splunk Platform (Enterprise/Cloud) users understand the difference between this EKS add-on and other available Splunk Enterprise/Cloud Add-ons.
2. Remove the confusion from the naming as this is not a Otel Collector Add-on but rather Otel Collector as an EKS add-on.

**Requirements**

- [X] Content follows Splunk guidelines for style and formatting.
- [X] You are contributing original content.

**Describe the change**

Enter a description of the changes, why they're good for the Observability Cloud documentation, and so on.

If this contribution is time sensitive, tell us when you'd like this PR to be merged.

### CODE REVIEW

File: gdi/opentelemetry/install-k8s-addon-eks.rst [hunk 1]
```diff
@@ -1,34 +1,32 @@
 .. _install-k8s-addon-eks:
 
 *******************************************************************************
-Splunk Distribution of the OTel Collector Add-on for AWS EKS
+Splunk Distribution of the OpenTelemetry Collector as EKS Add-on 
 *******************************************************************************
 
 .. meta::
-    :description: AWS Add-on for EKS
+    :description: Kubernetes (EKS Add-on)
```
Comment (jvoravong) [inline gdi/opentelemetry/install-k8s-addon-eks.rst:8]: Naming things is hard. I'll offer my input and some of the future products we will (likely) end up supporting. Let's collaborate as needed. 
```suggestion
    :description: Kubernetes via the EKS Add-on installation
```

We will have very similar to support in the future to support which might be named like this.
- Kubernetes via the GCP Add-on installation
- Kubernetes via the Azure Add-on installation

File: gdi/opentelemetry/install-k8s-addon-eks.rst [hunk 2]
```diff
@@ -1,34 +1,32 @@
 .. _install-k8s-addon-eks:
 
 *******************************************************************************
-Splunk Distribution of the OTel Collector Add-on for AWS EKS
+Splunk Distribution of the OpenTelemetry Collector as EKS Add-on
```
Comment (jvoravong) [inline gdi/opentelemetry/install-k8s-addon-eks.rst:4]: We should sate AWS in some form or manner on this page (Add-on for AWS EKS). I'll leave this up to the Docs team.

Comment (aurbiztondo-splunk) [reply gdi/opentelemetry/install-k8s-addon-eks.rst:4]: On it!

**Review by aurbiztondo-splunk — APPROVED** [review]

**Comment by aurbiztondo-splunk** [comment]
Thanks for the PR, @chentaow-splunk and @jvoravong! Made super minor edits, and publishing!

**Pull request merged**